sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/statsEstimation/BasicStatsPlanVisitor.scala

```diff
@@ -81,9 +81,9 @@ object BasicStatsPlanVisitor extends LogicalPlanVisitor[Statistics] {
     ProjectEstimation.estimate(p).getOrElse(fallback(p))
   }
 
-  override def visitRepartition(p: Repartition): Statistics = default(p)
+  override def visitRepartition(p: Repartition): Statistics = fallback(p)
```

Comment:
Q: we need to fall back into the size-based one instead of just calling `p.child.stats`?

Reply:
For better maintenance, if `BasicStatsPlanVisitor` and `SizeInBytesOnlyStatsPlanVisitor` have the same implementation, then fallback. For example `visitLocalLimit`:
https://github.com/apache/spark/blob/5553429728859472e3babd507163fba6c9fddfb9/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/statsEstimation/BasicStatsPlanVisitor.scala#L76

Reply:
Ah, I see. It's trivial and sgtm.